openpype/pipeline/colorspace.py

```diff
@@ -206,8 +251,9 @@ def validate_imageio_colorspace_in_config(config_path, colorspace_name):
     return True
 
 
+# TODO: remove this in future - backward compatibility
 def get_data_subprocess(config_path, data_type):
-    """Get data via subprocess
+    """[Deprecated] Get data via subprocess
```

Comment:
Anything marked deprecated should have a deprecated function decorator added so that whenever it gets called it prints the deprecation message including information what the replacement function should be if that exists.

Reply:
done ;)